contracts/interchain-token-service/src/msg.rs

```diff
@@ -31,6 +31,18 @@ pub enum ExecuteMsg {
     #[permission(Specific(gateway))]
     Execute(AxelarExecutableMsg),
 
+    /// Registers an existing ITS token with the hub. This is useful for tokens that were deployed
+    /// before the hub existed and have operated in p2p mode. Both instance_chain and origin_chain
+    /// must be registered with the hub.
+    #[permission(Elevated)]
```

Comment:
We should consider adding another role for these methods that's separate from the admin role, to make the migration easier to perform. It doesn't modify existing token instances so the risk is lower

Reply:
the roles are part of the permission control macro, and there's only two right now, admin and governance. What would the third role be? Operator?

Also, I'd like to push such a change to a separate PR.

Reply:
we can use the `Specific` modifier for custom permission checks, but as @cjcobb23 said, that should be done in a separate PR if necessary

Reply:
yeah, we should add a `Specific(operator)`. This operator role will be useful for other things like setting a rate limit in the future. And yes, it should be added in a separate PR